core/src/main/java/hudson/model/Computer.java

```diff
@@ -731,8 +731,10 @@ public void setTemporarilyOffline(boolean temporarilyOffline, OfflineCause cause
      */
     @Exported
     public String getIcon() {
-        if(isOffline())
+        if(getChannel() == null)
             return "computer-x.png";
+        else if (isTemporarilyOffline())
+            return "computer-u.png";
```

Comment:
Maybe an off-topic, but using longer self-describing file names could be helpful